feat(content): Add additional jobs to the Deep to improve vibe, help find side quests (#6145)
endless-sky/endless-sky: data/human/deep jobs.txt

	on complete
		dialog phrase "academic conference recording payment"
		payment 5000 1000

phrase "secure package pickup"
	word
		`As instructed, you`
		`You`
	word
		` `
	word
		`collect from a secure baggage counter`

> 		`take possession from a security desk`

Author: You know when you formally sign to receive a package. That's when you "take possession".

> 		`take possession from a security desk`

Reviewer: Apologies, I've never heard of that phrase used like that before.

	word
		` `
	word
		`in a quiet part of the spaceport`
		`near your landing site`
	word
		` `
	word
		`and pick up the secure package. It is a large box on a built-in secure antigrav trolley.`
		`and pick up the secure package. It is a small box with a complex, electronic combination lock.`
		`and pick up the secure package. It is a soft but tightly wrapped package that makes a disquieting sound if you squeeze it.`

phrase "secure package dropoff payment"
	word
		`As instructed, you deliver the package to`
	word
		` `
	word
		`the reception counter of a technology firm.`
		`a bored-looking spaceport security officer.`
		`an elaborate curiosities store.`

> 		`a plain clothes officer who receives it with minimal ID confirmation.`

Author: They're security, not police.

		`an old lady waiting on a bench.`
		`another ship captain waiting on the docks.`
	word
		` `
	word
		`Your bank account immediately notifies you that the agreed-upon payment of <payment> has been deposited. You wonder how they knew.`

mission "Secure Transport Job (Secret) [0]"
	name "Secure delivery to <planet>"
	job
	repeat
	description "Deliver a secure package of supplies to <destination>. Payment is <payment>."
	cargo "secure package" 1
	to offer

> 		random < 25

Author: It is meant to be reasonably common on the job board to give the area an integrated feeling. It implies that professional travel between worlds is common in the Deep, compared to elsewhere.

	on fail
		"reputation: Deep Security" -= 2
	on abort
		"reputation: Deep Security" -= 5

mission "Secure Transport Job (Secret) [4]"
	name "Secure delivery to <planet>"
	job
	repeat
	description "Deliver a secure package of supplies to <destination>. Payment is <payment>."
	cargo "secure package" 1
	to offer
		random < 20
		not "deep security reveal"
		"reputation: Deep Security" >= -50
	source
		attributes "deep"
		not attributes "station" "moon"
	destination
		attributes "deep"
		not planet "Valhalla"
		not attributes "station" "moon"
	on accept
		dialog phrase "secure package pickup"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment 50000 10000
		"reputation: Deep Security" += 0.5
		"secure deep package" ++
		dialog phrase "secure package dropoff payment"
	on fail
		"reputation: Deep Security" -= 2
	on abort
		"reputation: Deep Security" -= 5

mission "Deep Security Package Reveal"
	minor
	invisible
	source
		attributes "deep"
		not attributes "station" "moon"
	to offer
		"reputation: Deep Security" >= 100
		"secure deep package" >= 30
		"mystery cube" >= 5
		not "deep security reveal"
		or
			not "chosen sides"
			has "main plot completed"
	on offer

> 		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`

Author: I believe the original is correct in terms of the past tense of the log entry.

> 		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`

Reviewer: Not a big deal either way. Feel free to ignore this suggestion.

		conversation

> 			`As you walk away from a food stand in the spaceport a rather young man in active wear steps away from the wall and smoothly sidles up next to you between one step and the next.`

Reviewer: What does 'between one step and the next' mean in this context?

Author: It means literally what it says.

			`	His words come out very quickly as if they'd been heavily rehearsed and, looking around, no one appears to be paying any attention to either of you.`
			choice
				`	"Sure. Who are you?"`
					goto who
				`	"Sorry, but I'm busy right now."`
			`	"Uh, I was coming to thank you if that helps?" he says.`
			choice
				`	"Oh. What is it?"`
				`	"Sorry, maybe next time."`
					decline
			
			label who
			`	He continues walking and speaks without looking directly at you. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I'm uh, well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
			choice
				`	"Work is work."`
					goto blindacceptance
				`	"Ah, yes."`
					goto blindacceptance
				`	"Do all the secure packages belong to Deep Security?"`
			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
			label blindacceptance

> 			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion and it seems like you qualified. You passed the background check at any rate.`

Reviewer: ```suggestion
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion and it seems like you qualified. You passed the background check at any rate.`
```

Author: I feel like @W1zrad would come along and tell me to take that one back out again?

Reviewer 2: That's a spot where it's needed - you got two independent clauses so you need a comma. You _could_ stretch to keep it in since it's dialogue though.

Author: The comma wizard has spoken!

> 			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion and it seems like you qualified. You passed the background check at any rate.`

Author: I don't believe the second comma is warranted.

> 			`	"So at any rate, I'm authorized to give you a one time offer to join that list. Don't worry, I'm not asking anything of your right now."`

Reviewer: ```suggestion
			`	"So at any rate, I'm authorized to give you a one time offer to join that list. Don't worry, I'm not asking anything of you right now."`
```

			`	Belatedly he seems to do a double-check that no one is actively listening to the conversation as you walk. "So, you interested?"`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
				`	"You said 'at any rate' twice."`
			`	He looks away sheepishly. "Er, yes, sorry about that. The question stands though."`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
			label agree

> 			`	"Fantastic," he says, suddenly grinning widely enough to see in your periphery. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`

Reviewer: ```suggestion
			`	"Fantastic," he says, suddenly grinning wide enough to see in your periphery. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
```
Does 'see in your periphery' mean that you can see his smile out of the corner of your eye? I'm a bit confused as to what the phrase is supposed to mean.

Author: Yes that is exactly what it means.

Reviewer: I feel like a moron asking this but wouldn't this imply that the player isn't looking directly at Wayde? This strikes me as an odd turn of phrase.

Author: It has him walking beside you, you who have just got food from a food stand. I don't know about you, but I don't generally look directly at the person I'm taking to if we're walking alongside, and especially not if I have food.

			apply
				set "deep security reveal"

> 			`	Agent Wayde jogs off ahead of you, flicking a subtle salute your direction just before he turns a corner and disappears.`

Author: He's young and you're a captain, just for fun, as a formal wave, enthusiasm for salutes, respect, out of habit... like really take your pick.

Also, any motion can be subtle if it aligns sufficiently well with another action you'd be expected to do, so that only the person it's aimed at is likely to take notice.
e.g. A jogging person might push their hair back off their face, if you're not looking right at them a quick salute could be made a part of that action and it would not jump out as conspicuous to someone who was merely walking by and not explicitly watching the jogger.

		not attributes "station" "moon"
	destination
		attributes "deep"
		not planet "Valhalla"
		not attributes "station" "moon"
	on accept
		dialog phrase "secure package pickup"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment 80000 15000
		"reputation: Deep Security" += 0.5
		dialog phrase "secure package dropoff payment"
	on fail
		"reputation: Deep Security" -= 2
	on abort
		"reputation: Deep Security" -= 5

mission "Secure Transport Job (Deep Security) [1]"
	name "Deep Security delivery to <planet>"
	job
	repeat
	description "Deliver a Deep Security package to <destination>. Payment is <payment>."
	cargo "deep security package" 1
	to offer
		random < 20
		has "deep security reveal"
		"reputation: Deep Security" >= 100
	source
		attributes "deep"
		not attributes "station" "moon"
	destination
		attributes "deep"
		not planet "Valhalla"
		not attributes "station" "moon"
	on accept
		dialog phrase "secure package pickup"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment 80000 15000
		"reputation: Deep Security" += 0.5
		dialog phrase "secure package dropoff payment"
	on fail
		"reputation: Deep Security" -= 2
	on abort

> 		"reputation: Deep Security" -= 5

Reviewer: What is the difference between these two missions? I'm rather unfamiliar with mission syntax so apologies if I'm missing something obvious here.

Author: Repeating jobs lets more than one instance of them appear on the boards at the same time.

mission "Deep Consultants"
	minor
	source
		attributes "deep"
		attributes "urban"
		not attributes "station" "moon"
		not planet "Valhalla"
	destination "Valhalla"
	to offer
		"reputation: Deep Security" >= 100
		"secure deep package" >= 30
		"mystery cube" >= 5
		has "deep security reveal"
		or
			not "chosen sides"
			has "main plot completed"
	passengers 2
	deadline 2 1
	on offer
		conversation
			`Strolling through the spaceport you are suddenly accosted by a young and very breathless woman. "Please, Captain," she wheezes, "I'm so glad I caught you."`

> 			`	After drawing in a deep breath she stands up straighter and gestures to someone some distance away, "Captain I'm sorry. My colleague Julie and I are in dire need of urgent transport. Can you help?"`

Reviewer: ```suggestion
			`	After drawing in a deep breath she stands up straighter and gestures to someone some distance away. "Captain, I'm sorry, but my colleague Julie and I are in dire need of urgent transport. Can you help?"`
```

> 			`	As she catches her breath some more, her older colleague hustles over; bags in tow.`

Reviewer: ```suggestion
			`	As she catches her breath some more, her older colleague hustles over, bags in tow.`
```

			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"Where do you need to go?"`

> 				`	"Why is so urgent?"`

Reviewer: ```suggestion
				`	"Why is it so urgent?"`
```

> 			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar and my assistant and I were held back with our duties and missed our booked transport. We're in dire need of returning to Valhalla for several important business meetings but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`

Reviewer 3: ```suggestion
			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar and my assistant and I were held back with our duties and missed our booked transport. We're in dire need of passage to Valhalla for several important business meetings, but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`
```

			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"Who's your employer?"`
			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually but at the moment, indirectly, Deep Security."`
			`	She leans in slightly closer and lowers her voice. "We have access to their list of trustworthy pilots for emergencies. You'll be paid quite well."`
			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"I'm sorry, but I really can't help you right now."`
					decline
			label sure
			`	"Lead right on then," says Julie.`
				accept
	on complete
		log "Minor People" "Julie Gillar" `A consultant who worked with Deep Security. She requested emergency transport from you and showed you how to make yourself available to future consultants.`
		payment 350000
		set "deep security consultants"
		conversation

> 			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring you <payment>, she points out how you can change your settings with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`

Reviewer 3: ```suggestion
			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring <payment> into your account, she helps you change your job board perferences with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`
```

Author: 'preferences' is misspelled, but otherwise fair

Reviewer 3: derp

Reviewer 3: feel free to mark as resolved if you've changed it without introducing that error. :)

> 			`	"Thank you Captain <last>, you've been most helpful. Do keep an eye out in the spaceports around here though, Deep Security is notorious for keeping us longer than we booked for. There's often someone stuck."`

Reviewer 3: ```suggestion
			`	"Thank you Captain <last>, you've been most helpful. Do keep an eye out in the spaceports around here though. Deep Security is notorious for keeping us longer than we expected. There's often someone stuck and in need of rapid transport off-world."`
```

Author: I personally would still keep 'booked for'.

Reviewer 3: But it's not grammatically correct. You don't book for something. You book a flight or a hotel; you don't book for a flight or a hotel.

Author: I don't see how?
You can book a hotel _for_ longer than you need, so you can be kept longer than you booked for no?

Reviewer 2: How about "longer than _we're_ booked for"? Dunno about "booked for" by itself, but  one can say "we're booked for a stay at a hotel". If you've got the "are" in front of it, it seems right to me.
I'd do the suggestion myself, but mobile github is mean to me

phrase "stranded consultant pickup"
	word
		`In accordance with the lodging request, you`
		`Despite the short notice, you`
	word
		` `
	word
		`meet the consultants at their hotel accommodation`
		`collect the consultants from the security station`
		`fish the consultants out of a seedy bar`
		`rescue the consultants from some overbearing companions`
	word
		` `
	word
		`in a quiet part of the spaceport`
		`on the other side of town`
		`in a raucous quarter just outside the port`
	word
		` `
	word
		`and escort them on board.`

phrase "stranded consultant dropoff payment"
	word

> 		`Your consultant passengers thank you as you deliver them to their destination.`

Reviewer 3: ```suggestion
		`Your passengers thank you as you deliver them to their destination.`
```

(emphasizing that they are consultants seems unnecessary here)

Author: What about "The consultants thank you" instead?

Reviewer 3: That would be fine.

	word
		` `
	word

> 		`You find out only after that they left behind a soiled pair of boots.`

Reviewer 3: ```suggestion
		`You find out only afterwards that they left behind a soiled pair of boots.`
```

		`Afterwards you find a tie left hanging on a door handle.`

> 		`You kick crumpled documents that you find into the incinerator without looking at them.`

Reviewer: ```suggestion
		`You kick crumpled documents that you find after their trip into the incinerator without looking at them.`
```

> 		`You kick crumpled documents that you find after their trip into the incinerator without looking at them.`

Reviewer 3: ```suggestion
		`Later, you find some crumpled documents and wisely decide to toss them into the ship's incinerator without looking at them.`
```

	destination
		attributes "deep"
		not planet "Valhalla"
		not attributes "station" "moon"
	on accept
		dialog phrase "secure package pickup"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment 50000 10000
		"reputation: Deep Security" += 0.5
		"secure deep package" ++
		dialog phrase "secure package dropoff payment"
	on fail
		"reputation: Deep Security" -= 2
	on abort
		"reputation: Deep Security" -= 5

mission "Deep Security Package Reveal"
	minor
	invisible
	source
		attributes "deep"
		not attributes "station" "moon"
	to offer
		"reputation: Deep Security" >= 100
		"secure deep package" >= 30
		"mystery cube" >= 5
		not "deep security reveal"
		or
			not "chosen sides"
			has "main plot completed"
	on offer
		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`
		conversation
			`As you walk away from a food stand in the spaceport, a rather young man in activewear steps away from the wall and smoothly sidles up next to you, between one step and the next.`
			`	"Hello Captain. Please don't be alarmed, do you have a moment?"`

> 			`	His words come out very quickly as if they'd been heavily rehearsed. Glancing around, so far as you can tell, no one appears to be paying any attention to either of you.`

Author: Nah I get it, I know how it works, I'm just bad at sticking to a tense.

			choice
				`	"Work is work."`
					goto blindacceptance
				`	"Ah, yes."`
					goto blindacceptance
				`	"Do all the secure packages belong to Deep Security?"`
			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
			label blindacceptance
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
			`	Belatedly he seems to do a double-check that no one is actively listening to the conversation as you walk. "So, you interested?"`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline

> 				`	"You said 'at any rate' twice."`

Author: Uh huh, so can I disregard that one?

			`	He looks away sheepishly. "Er, yes, sorry about that. The question stands though."`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
			label agree

> 			`	"Fantastic," he says, suddenly grinning widely enough to be seen in your periphery as you walk along. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`

Reviewer 2: This spot in particular doesn't seem consistent with Wayde's characterization earlier in the conversation. He seems to be trying to hide his [eagerness, enthusiasm] and remain discreet until now. I can get him being a little more excited when the player agrees, but this feels a bit overboard - a huge grin feels almost like he's "breaking cover".

Author: Ah yes, now I see the problem with it. What do you think of this?
```suggestion
			`	"Fantastic," he says, quickly modulating his voice as someone briefly looks up at you from their phone. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
```

Reviewer 2: That's an angle I wouldn't have gone for, but certainly not bad. Works for me!

			apply
				set "deep security reveal"

> 			`	Agent Wayde flashes you a smile and jogs off ahead. Pushing his hair back out his eyes he flicks a subtle salute your direction just before he turns a corner and disappears.`

Reviewer 2: ```suggestion
			`	Agent Wayde flashes you a smile and jogs off ahead. Pushing his hair back out his eyes, he flicks a subtle salute your direction just before he turns a corner and disappears.`
```

> 			`As you walk away from a food stand in the spaceport, a rather young man in activewear steps away from the wall and smoothly sidles up next to you, between one step and the next.`
> 			`	"Hello Captain. Please don't be alarmed, do you have a moment?"`
> 			`	His words come out very quickly as if they've been heavily rehearsed. Glancing around, so far as you can tell, no one appears to be paying any attention to either of you.`
> 			choice
> 				`	"Sure. Who are you?"`
> 					goto who
> 				`	"Sorry, but I'm busy right now."`
> 			`	"Uh, I was coming to thank you if that helps?" he says.`
> 			choice
> 				`	"Oh. What is it?"`
> 				`	"Sorry, maybe next time."`
> 					decline
> 			
> 			label who
> 			`	He continues walking and speaks without looking directly at you. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I'm uh, well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
> 			choice
> 				`	"Work is work."`
> 					goto blindacceptance
> 				`	"Ah, yes."`
> 					goto blindacceptance
> 				`	"Do all the secure packages belong to Deep Security?"`
> 			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
> 			label blindacceptance
> 			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
> 			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
> 			`	Belatedly, he seems to do a double-check that no one is actively listening to the conversation as you walk. "So, you interested?"`
> 			choice
> 				`	"Oh sure, I'm willing to help."`
> 					goto agree
> 				`	"Sorry, but I don't think I want to get that involved."`
> 					decline
> 				`	"You said 'at any rate' twice."`
> 			`	He looks away sheepishly. "Er, yes, sorry about that. The question stands though."`
> 			choice
> 				`	"Oh sure, I'm willing to help."`
> 					goto agree
> 				`	"Sorry, but I don't think I want to get that involved."`
> 					decline
> 			label agree
> 			`	"Fantastic," he says, quickly modulating his voice as someone briefly looks up at you from their phone. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
> 			apply
> 				set "deep security reveal"
> 			`	Agent Wayde flashes you a smile and jogs off ahead. Pushing his hair back out his eyes, he flicks a subtle salute your direction just before he turns a corner and disappears.`

Author: @Pointedstick it seems we're starting from very different baseline expectations so I sought specific feedback on the Discord. No one saw any specific issue with Julie so I have focused on a rework of Agent Wayde's interaction where I was able to get some useful insight. Hopefully this satisfies the bulk of your concerns?

```suggestion
			`As you turn to leave a food stand in the spaceport, a rather young man in activewear waves you over to a bench off to the side of the main drag.`
			choice
				`	(Go over to him.)`
					goto bench
				`	(Keep walking.)
			`	When you go to ignore him he gives a quick, sharp, whistle and briefly flashes a deep security badge at you. He immediately covers this with a friendly wave for any nearby eyes, but it is clear that he won't be ignored.`
			label bench
			`	When you approach the bench he says, "Hello Captain. Please, have a seat and allow me to take a moment of your time."`
			`	His manner is very rehearsed, but he seems quite intent and assured in his request.`
			choice
				`	"Sure. Who are you?" (Sit down.)`
					goto who
				`	"Sorry, but I'm busy right now."`
			`	"To be clear, I was here thank you if that helps?" he says.`
			choice
				`	"Oh. What is it?" (Sit down.)`
				`	"Sorry, maybe next time."`
					decline
			label who
			`	He smiles at you before directing his attention back to those passing by as he speaks. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I... well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
			choice
				`	"Work is work."`
					goto blindacceptance
				`	"Ah, yes."`
					goto blindacceptance
				`	"Do all the secure packages belong to Deep Security?"`
			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs, giving you a measured look.`
			label blindacceptance
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
			`	He stops keeping an eye on passers-by for a moment to look directly at you. "So, you interested?"`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
				`	"You said 'at any rate' twice."`
			`	His expression twitches slightly but he carries on with a chuckle. "Er, good catch. The question stands though."`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
			label agree
			`	"Excellent," he says, standing up and stretching as if concluding a friendly conversation. He turns back to you with a smile, "I'll add you to the list as soon as I get back to the office then. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
			apply
				set "deep security reveal"
			`	Agent Wayde appears to flick a subtle salute your direction, or perhaps merely tips an imaginary cap. "Good day Captain, I look forward to working with you in the future."`
			`	With a quick motion he flicks his hair back out his eyes and jogs away, sending one last wave your direction just before he turns a corner and disappears.`
```

Reviewer 3: That's better for sure. I would suggest a number of additional minor changes afterwards (for example, remove "I... well "), but nothing that yould dramatically change Wayde.

		"reputation: Deep Security" += 0.5
		dialog phrase "secure package dropoff payment"
	on fail
		"reputation: Deep Security" -= 2
	on abort
		"reputation: Deep Security" -= 5

mission "Deep Consultants"
	minor
	source
		attributes "deep"
		attributes "urban"
		not attributes "station" "moon"
		not planet "Valhalla"
	destination "Valhalla"
	to offer
		"reputation: Deep Security" >= 100
		"secure deep package" >= 30
		"mystery cube" >= 5
		has "deep security reveal"
		or
			not "chosen sides"
			has "main plot completed"
	passengers 2
	deadline 2 1
	on offer
		conversation
			`Strolling through the spaceport, you are suddenly accosted by a young and breathless woman. "Please, Captain," she wheezes, "I'm so glad I caught you."`
			`	After drawing in a deep breath, she stands up straighter and gestures to someone some distance away. "Captain, I'm sorry, but my colleague Julie and I are in dire need of urgent transport. Can you help?"`
			`	As she catches her breath some more, her older colleague hustles over, bags in tow.`
			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"Where do you need to go?"`
				`	"Why is it so urgent?"`
				`	"Who are you?"`

> 			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar and my assistant and I were held back with our duties and missed our booked transport. We're in dire need of passage to Valhalla for several important business meetings, but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`

Reviewer 2: ```suggestion
			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar. My assistant and I were held back with our duties and missed our booked transport. We're in dire need of passage to Valhalla for several important business meetings, but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`
```
Personal preference - up to you.

			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"Who's your employer?"`

> 			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually but at the moment, indirectly, Deep Security."`

Reviewer 2: ```suggestion
			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually, but at the moment, indirectly, Deep Security."`
```

			`	She leans in slightly closer and lowers her voice. "We have access to their list of trustworthy pilots for emergencies. You'll be paid quite well."`
			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"I'm sorry, but I really can't help you right now."`
					decline
			label sure
			`	"Lead right on then," says Julie.`
				accept
	on complete
		log "Minor People" "Julie Gillar" `A consultant who worked with Deep Security. She requested emergency transport from you and showed you how to make yourself available to future consultants.`
		payment 350000
		set "deep security consultants"
		conversation

> 			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring <payment> into your account, she helps you change your job board preferences with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`

Reviewer 2: ```suggestion
			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark, she thanks you for your assistance. In addition to transferring <payment> into your account, she helps you change your job board preferences with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`
```

			`	"Thank you Captain <last>, you've been most helpful. Do keep an eye out in the spaceports around here though. Deep Security is notorious for keeping us longer than we booked for. There's often someone stuck and in need of rapid transport off-world."`

phrase "stranded consultant pickup"
	word
		`In accordance with the lodging request, you`
		`Despite the short notice, you`
	word
		` `
	word
		`meet the consultants at their hotel accommodation`
		`collect the consultants from the security station`
		`fish the consultants out of a seedy bar`
		`rescue the consultants from some overbearing companions`
	word
		` `
	word
		`in a quiet part of the spaceport`
		`on the other side of town`
		`in a raucous quarter just outside the port`
	word
		` `
	word
		`and escort them on board.`

phrase "stranded consultant dropoff payment"
	word
		`The consultants thank you as you deliver them to their destination.`
	word
		` `
	word

> 		`You find out only afterwards that they left behind a soiled pair of boots.`

Reviewer 2: ```suggestion
		`You find out only afterward that they left behind a soiled pair of boots.`
```
Small American vs. British English thing. The general rule for these is take off the "s" for the American version (like toward vs. towards).

> 		`Afterwards you find a tie left hanging on a door handle.`

Reviewer 2: ```suggestion
		`Afterward, you find a tie left hanging on a door handle.`
```

		dialog phrase "stranded consultant pickup"
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment 200000 35000
		"reputation: Deep Security" += 0.5
		dialog phrase "stranded consultant dropoff payment"
	on fail
		"reputation: Deep Security" -= 4
	on abort
		"reputation: Deep Security" -= 10

mission "Transport Consultants (Deep Security) [1]"
	name "Transport Deep Security consultants to <planet>"
	job
	repeat
	description "Transport consultants whose schedules have been ruined by Deep Security to <destination>. Payment is <payment>."
	passengers 1 8
	to offer
		random < 20
		has "deep security consultants"
		"reputation: Deep Security" >= 100
	source
		attributes "deep"
		not attributes "station" "moon"
	destination
		attributes "deep"
		attributes "urban"
		not attributes "station" "moon"
	on accept
		dialog phrase "stranded consultant pickup"
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment 250000 40000
		"reputation: Deep Security" += 0.5
		dialog phrase "stranded consultant dropoff payment"
	on fail
		"reputation: Deep Security" -= 4
	on abort
		"reputation: Deep Security" -= 10

mission "Urgent Deep Sec Package"

> 	name "Rush delivery of a Deep Security package to <planet>"

Reviewer 2: Some of these job names seem especially long. Do they display okay in-game?

Author: I honestly haven't tested. I reckon I could revise that to "Deliver a Deep Sec package to <planet>" which I'm fairly sure would fit.

Reviewer 3: Could be shortened to something like `Rush Deep Security Delivery to <planet>`

		not attributes "station" "moon"
	destination
		attributes "deep"
		not planet "Valhalla"
		not attributes "station" "moon"
	on accept
		dialog phrase "secure package pickup"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment 50000 10000
		"reputation: Deep Security" += 0.5
		"secure deep package" ++
		dialog phrase "secure package dropoff payment"
	on fail
		"reputation: Deep Security" -= 2
	on abort
		"reputation: Deep Security" -= 5

mission "Deep Security Package Reveal"
	minor
	invisible
	source
		attributes "deep"
		not attributes "station" "moon"
	to offer
		"reputation: Deep Security" >= 100
		"secure deep package" >= 30
		"mystery cube" >= 5
		not "deep security reveal"
		or
			not "chosen sides"
			has "main plot completed"
	on offer
		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`
		conversation
			`As you turn to leave a food stand in the spaceport, a rather young man in activewear waves you over to a bench off to the side of the main drag.`
			choice
				`	(Go over to him.)`
					goto bench
				`	(Keep walking.)`

> 			`	When you go to ignore him he gives a quick, sharp, whistle and briefly flashes a deep security badge at you. He immediately covers this with a friendly wave for any nearby eyes, but it is clear that he won't be ignored.`

Author: I feel like the "go to" is important, like its a much more on-the-ball, instant like "no I saw you look at me don't you dare start to walk away" snappy response.

> 			`	When you go to ignore him he gives a quick, sharp, whistle and briefly flashes a deep security badge at you. He immediately covers this with a friendly wave for any nearby eyes, but it is clear that he won't be ignored.`

Author: ```suggestion
			`	When you go to ignore him he gives a quick, sharp, whistle and briefly flashes a Deep Security badge at you. He immediately covers this with a friendly wave for any nearby eyes, but it is clear that he won't be ignored.`
```

			label bench
			`	When you approach the bench he says, "Hello Captain. Please, have a seat and allow me to take a moment of your time."`
			`	His manner is very rehearsed, but he seems quite intent and assured in his request.`
			choice
				`	"Sure. Who are you?" (Sit down.)`
					goto who
				`	"Sorry, but I'm busy right now."`

> 			`	"To be clear, I was here thank you if that helps?" he says.`

Author: Seems wordier than necessary at the end, though I feel ya for the general intent.

> 			`	"To be clear, I was here thank you if that helps?" he says.`

Author: ```suggestion
			`	"To be clear, I'm here to thank you, if that helps?" he says.`
```

> 			`	"To be clear, I was here thank you if that helps?" he says.`

Reviewer 2: ```suggestion
			`	"To be clear, I'm here to thank you, if that helps." he says.`
```
Having the question mark there does help with tone a little, but it crosses too far into feeling unprofessional or obviously incorrect for my tastes.

			choice
				`	"Oh. What is it?" (Sit down.)`
				`	"Sorry, maybe next time."`
					decline
			label who

> 			`	He smiles at you before directing his attention back to those passing by as he speaks. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I... well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`

Reviewer 2: I actually prefer this in tandem with the previous comment you've (Grey) resolved. It is more standard to have just dialogue as a choice.

			choice

> 				`	"Work is work."`

Reviewer 4: ```suggestion
				`	"If it pays, it pays."`
```

					goto blindacceptance

> 				`	"Ah, yes."`

Reviewer 4: ```suggestion
				`	"I recall doing those jobs, yes."`
```

Author: Seems overly formal?

					goto blindacceptance
				`	"Do all the secure packages belong to Deep Security?"`
			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs, giving you a measured look.`
			label blindacceptance
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
			`	He stops keeping an eye on passers-by for a moment to look directly at you. "So, you interested?"`
			choice

> 				`	"Oh sure, I'm willing to help."`

Reviewer 4: ```suggestion
				`	"I'm willing to help."`
```

					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
				`	"You said 'at any rate' twice."`
			`	His expression twitches slightly but he carries on with a chuckle. "Er, good catch. The question stands though."`
			choice

> 				`	"Oh sure, I'm willing to help."`

Reviewer 4: ```suggestion
				`	"I'm willing to help."`
```

					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
			label agree

> 			`	"Excellent," he says, standing up and stretching as if concluding a friendly conversation. He turns back to you with a smile, "I'll add you to the list as soon as I get back to the office then. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`

Reviewer 4: ```suggestion
			`	"Excellent," he says, standing up and stretching as if concluding a friendly conversation. He turns back to you with a smile, "I'll add you to the list as soon as I get back to the office then. Your ID will also let you see which packages are ours specifically too. Oh, and again, thank you!"`
```

			apply
				set "deep security reveal"

> 			`	Agent Wayde appears to flick a subtle salute your direction, or perhaps merely tips an imaginary cap. "Good day Captain, I look forward to working with you in the future."`

Reviewer 4: ```suggestion
			`	Agent Wayde appears to flick a subtle salute your direction, or perhaps merely tips an imaginary cap. "Good day, captain. I look forward to working with you in the future."`
```

			`	As she catches her breath some more, her older colleague hustles over, bags in tow.`
			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"Where do you need to go?"`
				`	"Why is it so urgent?"`
				`	"Who are you?"`
			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar. My assistant and I were held back with our duties and missed our booked transport. We're in dire need of passage to Valhalla for several important business meetings, but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`
			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"Who's your employer?"`
			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually, but at the moment, indirectly, Deep Security."`
			`	She leans in slightly closer and lowers her voice. "We have access to their list of trustworthy pilots for emergencies. You'll be paid quite well."`
			choice
				`	"Sure, I'd be glad to help.`
					goto sure
				`	"I'm sorry, but I really can't help you right now."`
					decline
			label sure

> 			`	"Lead right on then," says Julie.`

Reviewer 4: ```suggestion
			`	"Lead right on then," says Julie. You point towards your ship, and the women move to board it.`
```

				accept
	on complete
		log "Minor People" "Julie Gillar" `A consultant who worked with Deep Security. She requested emergency transport from you and showed you how to make yourself available to future consultants.`
		payment 350000
		set "deep security consultants"
		conversation

> 			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark, she thanks you for your assistance. In addition to transferring <payment> into your account, she helps you change your job board preferences with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`

Reviewer 4: Model passengers in what way?

Author: Polite, clean, organised, respectful, etc.? They were ideal passengers from the player's perspective, I don't think it needs to be explicit.
